packages/events/examples/events_thread_safe.rs

```diff
@@ -0,0 +1,87 @@
+//! Example demonstrating thread-safe event usage.
+//!
+//! This example shows how to use Event<T> (thread-safe) in various scenarios,
+//! including cross-thread communication and different wrapper types.
+
+use std::rc::Rc;
+use std::sync::Arc;
+use std::thread;
+
+use events::once::Event;
+
+/// Helper function to assert that a type implements Send + Sync
```

Comment:
Useful in unit tests but a bit overboard in examples - let's keep it short and to the point.